### PR TITLE
sample(REPORT-366217): WPF .NET Core JSON extension sample

### PR DESCRIPTION
**Pull request by ArumugasamiMurugesan**
<table>
<tr>
<td>
Description:
</td>
<td>
WPF .NET Core JSON extension sample
</td>
</tr>
<tr>
<td>
Analysis:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Reason for not identifying earlier:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Is it a breaking issue?
</td>
<td>
No
</td>
</tr>
<tr>
<td>
Solution description:
</td>
<td>
WPF .NET Core JSON extension sample
</td>
</tr>
<tr>
<td>
Areas affected and ensured:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Test cases:
</td>
<td>
NA
</td>
</tr>
<tr>
<td>
Tested sample location:
</td>
<td>
NA
</td>
</tr>
</table>

### CODE REVIEW

File: WPF/WPF.NETCore-JSONExtension/BoldReports.DataExtensions.WebData/Base/CustomHttpClient.cs
Comment (ManoranjanRajendran) [inline WPF/WPF.NETCore-JSONExtension/BoldReports.DataExtensions.WebData/Base/CustomHttpClient.cs:1]: Change the name to common

File: WPF/WPF.NETCore-JSONExtension/ReportViewerWPFCore/App.config [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿<configuration>
+  <configSections>
+    <section name="ReportingExtensions" type="BoldReports.Configuration.Extensions,  BoldReports.WPF" allowLocation="true" allowDefinition="Everywhere" />
+  </configSections>
+  <ReportingExtensions>
+    <DataExtension>
+      <Extension Name="JSON" Assembly="BoldReports.Data.WebData" Type="BoldReports.Data.WebData.JSONExtension" />
```
Comment (ManoranjanRajendran) [inline WPF/WPF.NETCore-JSONExtension/ReportViewerWPFCore/App.config]: Add the all extension

**Comment by ArumugasamiMurugesan** [comment]
Hi @ManoranjanRajendran ,

Please find the JSON report output below in the WPF .NET Core sample.
image

Regards,
Arumugasami M

File: WPF/WPF.NETCore-Barcode-Extension/README.md [hunk 2]
```diff
@@ -1,24 +1,24 @@
...
-Enhance your WPF .NET Core applications' reporting capabilities by integrating barcode functionality using Bold Reports. Below are the steps to add a barcode extension to the Bold Reports Report Viewer in a WPF .NET Core application.
+Enhance your WPF .NET Core applications reporting capabilities by integrating barcode functionality using Bold Reports. Below are the steps to add a barcode extension to the Bold Reports Report Viewer in a WPF .NET Core application.
 
 ## Requirements
 
 Ensure you have the following prerequisites before proceeding:
 
 * Visual Studio 2019 or later
-* .NET 5.0 or later
+* .NET 5.0 or higher
```
Comment (ManoranjanRajendran) [inline WPF/WPF.NETCore-Barcode-Extension/README.md:10]: Change the sample to .net 6 or 8.

.Net5 don't have Long term support

**Review by ManoranjanRajendran — APPROVED** [review]
Changes Fine

File: WPF/WPF.NETCore-Barcode-Extension/README.md (hunk 2 above)
Comment (ArumugasamiMurugesan) [reply WPF/WPF.NETCore-Barcode-Extension/README.md:10]: Resolved the changes in the Pull Request 12

**Pull request merged**